WcaOnRails/app/views/avatars_mailer/notify_user_of_avatar_removal.html.erb

```diff
@@ -0,0 +1,12 @@
+<p>Hello <%= @user.name %></p>
+
+<p>Your avatar has been removed by <%= @remover %>.</p>
+
+<% if @reason.present? %>
+  <p>
+    Here's the reason:
+    <blockquote><%= @reason %></blockquote>
+  </p>
+<% end %>
+
+<p>Please provide a more suitable picture.</p>
```

Comment:
A signature at the end would be nice, with something like:
<p>
  Regards,
  WCA Results Team
</p>

Reply:
@danieljames-dj resolved in 0b2f9e39bedd207f183104d5b7a6151b79560a43🚀